src/ui.rs

```diff
@@ -619,9 +619,11 @@ impl<T> Snarl<T> {
         }
 
         if viewer.has_on_hover_popup(&self.nodes[node.0].value) {
-            r.on_hover_ui_at_pointer(|ui| {
-                viewer.show_on_hover_popup(node, &inputs, &outputs, ui, snarl_state.scale(), self);
-            });
+            if let Some(r) = r {
```

Comment:
The `r: Response` is returned from `r.context_menu`. Semantic of the returned response changed.
Previously `Response::context_menu` call was consuming `Response` and returning it unmodified.
Now it does not consume `Response` and returns `Option<InnerResponse>` associated with context menu interaction.
So it should not be used he, the old one should.

Reply:
Sorry did not notice anything that would be not working.